SPECS/nvidia-container-runtime/nvidia-container-runtime.spec

```diff
@@ -0,0 +1,63 @@
+Summary:      NVIDIA container runtime
+Name:         nvidia-container-runtime
+Version:      3.4.2
+Release:      2%{?dist}
+License:      ASL 2.0
+URL:          https://github.com/NVIDIA/nvidia-container-runtime
+Group:        Development Tools
+Vendor:       Microsoft Corporation
+Packager:     NVIDIA CORPORATION <cudatools@nvidia.com>
+Distribution: Mariner
+#Source0:     https://github.com/NVIDIA/${name}/archive/v%{version}.tar.gz
+Source0:      %{name}-%{version}.tar.gz
+
+BuildRequires: golang
+
+Requires: libseccomp
+
+%description
+Provides a modified version of runc allowing users to run GPU enabled
+containers.
+
+%prep
+%setup
+
+%build
+cd src/
+make build
+mkdir -p %{buildroot}%{_bindir}
+cp nvidia-container-runtime %{buildroot}%{_bindir}
+
+%install
+cd src
+install -m 755 nvidia-container-runtime %{buildroot}%{_bindir}/%{name}
+
+%files
+%license LICENSE
+%{_bindir}/nvidia-container-runtime
+
+%changelog
+* Wed Apr 21 2021 Joseph Knierman <jknierman@microsoft.com> 3.4.2-2
+- Initial CBL-Mariner import
```

Comment:
Please follow the changelog phrasing we've talked about while discussing our [spec guidelines](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/125/Introduction-to-SPEC-Files?anchor=spec-file-guidelines-and-best-practices) offline. Please don't forget to verify the license and mention it in the changelog as well with the phrase "License verified".

Reply:
Added in this change. The spec file was in the same repo as the source code so shoul also be under the ASL 2.0 licenses.